scrapy_poet/downloadermiddlewares.py

```diff
@@ -34,7 +34,7 @@
     PageParamsProvider: 700,
     RequestUrlProvider: 800,
     ResponseUrlProvider: 900,
-    ItemProvider: 1000,
+    ItemProvider: 2000,
```

Comment:
Increasing this value allows `ZyteApiProvider` _(as well as any other item provider in the future)_ to run before scrapy-poet's `ItemProvider`. This ensures any item dependency in page objects are available.